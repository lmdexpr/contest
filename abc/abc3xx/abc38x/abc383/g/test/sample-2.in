20 4
-5 3 4 -1 6 -2 13 -1 13 7 6 -12 3 -5 12 -6 -3 10 -15 -5
