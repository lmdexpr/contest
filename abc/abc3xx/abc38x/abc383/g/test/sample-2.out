32 45 57 52 22
